### PR TITLE
Replace list_t with wl_list

### PR DESCRIPTION
**Pull request by raazvvann**
partial work for #264

### CODE REVIEW

File: backend/multi/backend.c [hunk 1]
```diff
@@ -65,8 +65,8 @@ struct wlr_backend *wlr_multi_backend_create(struct wlr_session *session) {
 		return NULL;
 	}
 
-	backend->backends = list_create();
-	if (!backend->backends) {
+	wl_list_init(&backend->backends);
+	if (!&backend->backends) {
```
Comment (emersion) [inline backend/multi/backend.c]: I don't think this can fail.

Comment (raazvvann) [reply backend/multi/backend.c]: that's right. missed this

**Comment by raazvvann** [comment]
I am still working on replacing list_t with wl_list in wlr_wl_backend now. I'll let you know if I manage to do more on this.

**Comment by acrisci** [comment]
You don't need to free the lists.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**